app/settings.py

```diff
@@ -149,6 +149,10 @@ def utcoffset_or_fail(date_value, key):
     "ACCOUNT_SERVICE_BASE_URL", "https://surveys.ons.gov.uk"
 )
 
+ACCOUNT_SERVICE_BASE_URL_SOCIAL = os.getenv(
+    "ACCOUNT_SERVICE_BASE_URL_SOCIAL", "https://rh.ons.gov.uk"
+)
```

Comment:
Just needs to be a constant, as we would never set an env var for it given it is only used if you don't have a cookie.

Reply:
Removed the env var, was not sure about it so added just in case, thanks.